File: java/command/src/main/StockManager.java
package main;

public class StockManager {
    public String _name = "Laptop";
    private int _quantity = 10;

    public void Buy()
    {
        System.out.println("Stock: " + _name + " "+  _quantity +" bought ");
    }

    public void Sell()
    {
        System.out.println("Stock: " + _name + " "+  _quantity +" sold ");
    }
}

File: csharp/Command/StockManager.cs
namespace Command;

public class StockManager
{
    public string _name = "Laptop";
    private int _quantity = 10;

    public void Buy()
    {
        Console.WriteLine("Stock: {0}, {1} bought", _name, _quantity);
    }

    public void Sell()
    {
        Console.WriteLine("Stock: {0}, {1} sold", _name, _quantity);
    }
}